app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
```

Comment:
This is auto-generated by Pundit as a parent class for non-headless policies. It makes sense if you use Pundit in a certain way, but we chose a headless policy because of the unique characteristics of a step-by-step wizard, and because each stage is not a database-backed record itself, but rather an attribute value on User.